router/smoke/openshift_2/inventory-amq-interconnect-1-basic-to-router.yml

```diff
@@ -18,26 +18,15 @@ all:
         router_config: /etc/qpid-dispatch/qdrouterd.conf
         implementation: dispatch
         service: qdrouterd
+        ansible_connection: local
```

Comment:
That is needed in this case, cause otherwise ansible will try to run commands on a "router1" node (which does not exist).